alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestAcknowledgeAlertAction.kt

```diff
@@ -61,6 +63,8 @@ import java.time.Instant
  */
 class RestAcknowledgeAlertAction(settings: Settings, controller: RestController) : BaseRestHandler(settings) {
 
+    private val log: Logger = LogManager.getLogger(javaClass)
```

Comment:
Declare `log` outside the class to make it a singleton.

Reply:
Done.